WordPress/src/test/java/org/wordpress/android/ui/mysite/items/SiteItemsBuilderTest.kt

```diff
@@ -34,6 +35,7 @@ class SiteItemsBuilderTest {
 
     @Mock
     lateinit var quickStartType: QuickStartType
+    @Mock lateinit var jetpackFeatureRemovalOverlayUtil: JetpackFeatureRemovalOverlayUtil
```

Comment:
np: this new line-of-code could be adapted with newline after `@Mock` and a whitespace before and after it, to align with the updated code style in the test class 👍 

Same on line `50` we should align the new param with the other ones 🙏

Reply:
Hey ovi, 👋🏼 

Thanks for noticing it. I have updated the code to align the new param with other ones in [2db424a](https://github.com/wordpress-mobile/WordPress-Android/pull/17728/commits/2db424ae44f451ed026995a9f04e95e1ff97a712)